lib/Mojolicious/Plugin/DefaultHelpers.pm

```diff
@@ -196,6 +196,7 @@ sub _is_fresh {
 
 sub _json_exception {
   my ($c, $e) = @_;
+  $c->stash->{exception} = _is_e($e) ? $e : Mojo::Exception->new($e);
```

Comment:
`_is_e($e) ? $e : Mojo::Exception->new($e)` should probably be abstracted in a way so it gets free test coverage from somewhere else.

Reply:
Just pushed up, would you like me to write tests for it as well?

Reply:
By abstracting it out it should now be covered by existing tests automatically.